iot-hub/Samples/service/PnpServiceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -20,39 +20,39 @@ public class TemperatureControllerSample
 
         private readonly ServiceClient _serviceClient;
         private readonly RegistryManager _registryManager;
-        private readonly string _digitalTwinId;
+        private readonly string _deviceTwinId;
```

Comment:
If keeping consistency with msdocs (https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-digital-twin#device-twins-and-digital-twins), this should be renamed to _deviceId here and below